pkg/migration/controllers/migration.go

```diff
@@ -38,6 +38,9 @@ const (
 	// StorkMigrationReplicasAnnotation is the annotation used to keep track of
 	// the number of replicas for an application when it was migrated
 	StorkMigrationReplicasAnnotation = "stork.libopenstorage.org/migrationReplicas"
+	// StorkMigrationAnnotation is the annotation used to keep track of resources
+	// migrated by stork
+	StorkMigrationAnnotation = "stork.libopenstorage.org/storkMigration"
```

Comment:
`stork.libopenstorage.org/migrated` should be good. Don't need `stork` in the final key, it is in the first part of the annotation already.